packages/repo/Dockerfile.repo-rpm

```diff
@@ -1,7 +1,6 @@
-FROM centos:7
+FROM amazonlinux:2
 
-# Replace with ARM base image when building on ARM:
-#FROM arm64v8/centos:7
+# Replace with ARM base image when building on ARM
```

Comment:
I think we can actually drop that comment now, because `amazonlinux:2` has packages for both ARM and x86 on the same name (see https://hub.docker.com/_/amazonlinux/tags).